### DISCUSSION TITLE
i18n `structure` with alternative `multiple_folders`

### DISCUSSION THREAD

**Discussion by lorensansol** [StaticJsCMS/static-cms#969] [Ideas]
In the general i18n support configuration, 3 options are offered for the `structure`:

> Required and can be one of multiple_folders, multiple_files or single_file
> multiple_folders - persists files in `<folder>/<locale>/<slug>.<extension>`
> multiple_files - persists files in `<folder>/<slug>.<locale>.<extension>`
> single_file - persists a single file in `<folder>/<slug>.<extension>`

An alternative `multiple_folders` (maybe `single_folder`) that persists files in `<locale>/<folder>/<slug>.<extension>` would be great.

**Comment by KaneFreeman** [comment]
I've made an issue for this. #973
